internal/pkg/service/common/etcdop/op/join.go

```diff
@@ -61,7 +61,7 @@ func (v *JoinTo[R]) Do(ctx context.Context, client *etcd.Client) (R, error) {
 	return *v.result, nil
 }
 
-func (v *JoinTo[R]) MapResponse(ctx context.Context, response etcd.OpResponse) (result any, err error) {
+func (v *JoinTo[R]) MapResponse(ctx context.Context, response Response) (result any, err error) {
```

Comment:
Upraveny `MapResponse` interface, ktory sa pouziva, ked pridas operaciu do `TXN` a takto sa spracuje response.

`ForEachOp`, v dalsom commite, potrebuje pocas `MapResponse` volat `client`, preto som tento interface upravil, a pridal novy typ `Response`, ktory obsahuje aj to aky `client` a ake `optons` boli pouzite pri nacitani.